ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -577,6 +624,37 @@ -(WKWebView *)webView:(WKWebView *)webView createWebViewWithConfiguration:(WKWeb
     return nil;
 }
 
+#pragma mark - Networking
+- (void)makePostRequestWithUrl:(NSString *_Nonnull)urlString parameters:(NSString* _Nonnull)parameters completion:(RequestCompletion)completion {
+
+    NSData *parameterData = [parameters dataUsingEncoding:NSUTF8StringEncoding];
+    NSURL *url = [NSURL URLWithString:urlString];
+
+    NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:url];
+    [request addValue: @"application/x-www-form-urlencoded; charset=utf-8" forHTTPHeaderField:@"Content-Type"];
+    [request setHTTPMethod:POST_METHOD];
+    [request setHTTPBody:parameterData];
+
+    NSURLSession *session = [NSURLSession sharedSession];
+    NSURLSessionDataTask *task = [session dataTaskWithRequest:request
+                                            completionHandler:completion];
+    
+    [task resume];
+}
+
+#pragma mark - Keychain access
+- (void)saveDataFromJson:(NSDictionary* _Nonnull)json {
```

Comment:
It is not clear enough what this function is expecting/saving just by reading its name. Please consider changing it for something more descriptive